src/front-door/azext_front_door/_params.py

```diff
@@ -129,9 +129,9 @@ def load_arguments(self, _):
             c.argument('https_port', type=int, help='HTTPS TCP port number.')
             c.argument('weight', type=int, help='Weight of this endpoint for load balancing purposes.')
             c.argument('private_link_alias', help='The Alias of the Private Link resource. Populating this optional field indicates that this backend is \'Private\'.')
-            c.argument('private_link_resource_id', help='The Resource Id of the Private Link. Populating this optional field indicates that this backend is \'Private\'.')
+            c.argument('private_link_resource_id', options_list='--resource-id', help='The Resource Id of the Private Link. Populating this optional field indicates that this backend is \'Private\'.')
```

Comment:
It will cause breaking change for interface. Please add the old operation name.
```suggestion
            c.argument('private_link_resource_id', options_list=['--private-link-resource-id', '--resource-id'], help='The Resource Id of the Private Link. Populating this optional field indicates that this backend is \'Private\'.')
```